src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -521,6 +535,18 @@ class HMICapabilities {
    */
   virtual std::set<hmi_apis::FunctionID::eType>
   GetDefaultInitializedCapabilities() const = 0;
+
+  /**
+   * @brief Notified that response was received from requested interface
```

Comment:
@ydementieiev Who is notified that response was received?

Reply:
@AByzhynar   [done](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/2d5949fecc86315df964b15a2244ef309a0f87e4#diff-894b7bf70dcdcd402ffb670af6f4ff1fR540)